File: bwl.Network.ClientServer.FTDI/FtdiCable.vb
Public Class FTDICable
    Public Shared ReadOnly Property FtdiName = "Usb2UsbCable"

    Public Shared Function Find() As String()
        Dim result As New List(Of String)
        Dim ports = FTDIFunctions.GetFtdiPorts
        For Each port In ports
            If port.Description.Contains(FtdiName) Then
                Dim portName = FTDIFunctions.DetectFtdiSystemPortName(port)
                result.Add(portName.Trim)
            End If
        Next
        Return result.ToArray
    End Function

End Class

File: Bwl.Network.ClientServer.FTDI.Avalonia/FtdiCable.cs
using System.Collections.Generic;

namespace Bwl.Network.ClientServerMessaging
{

    public class FTDICable
    {
        public static string FtdiName { get; private set; } = "Usb2UsbCable";

        public static string[] Find()
        {
            var result = new List<string>();
            FTD2XX_NET.FTDI.FT_DEVICE_INFO_NODE[] ports = ClientServerMessaging.FTDIFunctions.GetFtdiPorts();
            foreach (var port in ports)
            {
                if (port.Description.Contains(FtdiName))
                {
                    string portName = ClientServerMessaging.FTDIFunctions.DetectFtdiSystemPortName(port);
                    result.Add(portName.Trim());
                }
            }
            return result.ToArray();
        }

    }
}